### DISCUSSION TITLE
Combined Docker Image

### DISCUSSION THREAD

**Discussion by Nixon506E** [mmcc-xx/BirdCAGE#26] [Ideas]
Hoping this could be set up within a single docker image so as to make it easier for users to install. Possibly include only the analyzer, backend, and frontend images with environment variables to determine which analyzer to use and where the redis server is located, etc.

**Comment by matthew73210** [comment]
I think docker compose is the way forward. Kinda defeats the purpose of docker if you have it as one whole image.

**Reply by mmcc-xx** [reply]
Yeah what he said... however, I think the docker compose could be made easier to manage. I find it a bit daunting, and I wrote it.

**Comment by matthew73210** [comment]
> Hoping this could be set up within a single docker image so as to make it easier for users to install. Possibly include only the analyzer, backend, and frontend images with environment variables to determine which analyzer to use and where the redis server is located, etc.

Btw, you can always do docker in docker or sysbox.
